### DISCUSSION TITLE
Calculating the mAP?

### DISCUSSION THREAD

**Discussion by SWRIGH211** [ultralytics/yolov5#2476] [Q&A]
Hi there, 
I'm trying to compare my custom trained yolo5s model against other models, however they do no state the mAP@.5  or the  mAP@.5:.95. only AP. My question is, how do you attain the AP of your model. I had a look at this medium explanation:

https://medium.com/@timothycarlen/understanding-the-map-evaluation-metric-for-object-detection-a07fe6962cf3

However im still quite confused on how to calculate it properly. 

Does anyone have a clearer simpler explanation, it would be much appreciated.

Many thanks in advance

**Comment by SWRIGH211** [comment]
its in relation to the The mean Average Precision (mAP) quantitative measure from PASCAL VOC 2010.

**Reply by glenn-jocher** [reply]
@SWRIGH211 VOC primary metric is mAP@0.5.

mAP computation code is here:
https://github.com/ultralytics/yolov5/blob/ed2c74218d6d46605cc5fa68ce9bd6ece213abe4/utils/metrics.py#L18-L31

**Reply by SWRIGH211** [reply]
thankyou makes sense
